js/views/harvester.vue

```diff
@@ -39,7 +47,7 @@
 import HarvestSource from 'models/harvest/source';
 import HarvestJob from 'models/harvest/job';
 import ItemModal from 'components/harvest/item.vue';
-import Vue from 'vue';
+import {harvest_enable_manual_run} from "../config";
```

Comment:
Maybe in a clean VueJS app :-D This app is an old VueJS v1, so I don't think we want to mess with the config :-D But in the new `udata-front` app yes it's better. This frontend should disappear in the near future in favor of `udata-front`.

Reply:
OK, gotcha

Reply:
On a second thought, even if we don't have a `vue-config.js` config file, could the app-absolute path import `{harvest_enable_manual_run} from "config";` work?
Since the other imports use that root path, this import should be the same way ideally

Reply:
Yes it works :-)